cryptography/polynomial/src/domain.rs

```diff
@@ -59,6 +65,9 @@ impl Domain {
             .invert()
             .expect("coset generator should not be zero");
 
+        let twiddle_factors = precompute_twiddle_factors(&generator, size);
+        let twiddle_factors_inv = precompute_twiddle_factors(&generator_inv, size);
```

Comment:
technically we could get the _inv version from the normal version -- though this would require an allocation each time we needed to do a IDFT